js/libs/form.js

```diff
@@ -225,6 +225,11 @@ Fliplet().then(function () {
               case 'flCheckbox': // There is no validation and value assignment as there is no access to checkbox options. This is implemented in the checkbox component.
               case 'flRadio': // There is no validation and value assignment as there is no access to radio options. This is implemented in the radio component.
 
+              case 'flSlider':
```

Comment:
I don't understand why this is necessary or why it's here.

This entire `switch` clause is about assigning `fieldData` from the data source to `field.value`.

Progress is reassigned back to the field in the `if (progress && !isEditMode)` part after this `switch` clause.